scripts/application-create-secrets-viai-client.sh

```diff
@@ -95,14 +95,19 @@ cp "${WORKING_DIRECTORY}/kubernetes/viai-camera-integration/secret_pubsub.yaml.t
 if [ -f "${SERVICE_ACCOUNT_KEY_PATH}" ]; then
   echo "Updating Pub/Sub credential..."
   SECRET_JSON_PATH="${SERVICE_ACCOUNT_KEY_PATH}"
-  base64 "$SECRET_JSON_PATH" >"$SECRET_JSON_PATH.tmp"
+  if is_linux; then
```

Comment:
Worth putting this in a function as you did with sed?

Reply:
updated